Exposing getIfPresent in the CaffeineCache (#26863)
quarkusio/quarkus: docs/src/main/asciidoc/cache.adoc

}
----

=== Retrieving a value if a key is present from a `CaffeineCache`

The cache value from a specific `CaffeineCache` can be retrieved if present as shown below.

> If the given key is contained in the cache, the returned `CompletableFuture` contains (existing or computed) future value to which the specified key is mapped to.

Reviewer: ```suggestion
If the given key is contained in the cache, the method will return the `CompletableFuture` the specified key is mapped to.
That `CompletableFuture` may be computing or may already be completed.
```

> Otherwise, the method with return the `null` value.

Reviewer: The second part of this suggestion isn't mandatory :smiley: 
```suggestion
Otherwise, the method will return `null`.
```

[source,java]
----
package org.acme.cache;

import javax.enterprise.context.ApplicationScoped;

import io.quarkus.cache.Cache;
import io.quarkus.cache.CacheName;
import io.quarkus.cache.CaffeineCache;

> import java.util.Set;

Author: review imports!
